modules/phase_field/src/ics/SmoothCircleBaseIC.C

```diff
@@ -137,8 +137,7 @@ SmoothCircleBaseIC::computeCircleValue(const Point & p, const Point & center, co
     }
 
     case ProfileType::TANH:
-      return (_invalue - _outvalue) * 0.5 *
-                 (std::tanh(libMesh::pi * (radius - dist) / _int_width) + 1.0) +
+      return (_invalue - _outvalue) * 0.5 * (std::tanh(2.0 * (radius - dist) / _int_width) + 1.0) +
```

Comment:
why this change?

Reply:
I see, you are probably defining the interface through the tangent at the mid-point and its intercepts with y=0 and y=1. What I did was matching the derivative at the mid point to the existing cosine interface profile. The previous method makes the interface with more consistent between the COS and TANH profile choices and is what I would prefer.

Reply:
![image](https://user-images.githubusercontent.com/202302/57804404-d3084b00-7717-11e9-85f6-eecd0e11fc8c.png)

Reply:
[Moelans 2008](https://journals.aps.org/prb/abstract/10.1103/PhysRevB.78.024113) calculated the equilibrium interface profile as this function for symmetric interfaces. That may just be for the free energy she proposed and not necessarily for all free energy functions, but it strikes me as more justifiable than the current form. If you prefer the pi we can keep it that way.

Reply:
Are you implying that Moelans' _l_ = `int_width` in your formulation? Because that is definitely not the case.

Reply:
Yeah. She calls it a GB width but it applies to any interface. Why is that not the case?

Reply:
Argh, I messed this up. You are right, the factor should be 2.0 to make it consistent with Moelans' lGb. Let's make that change. I hope it does not break too many inputs...